Makefile

```diff
@@ -4,14 +4,20 @@
 
 .PHONY: clean default docs help lint
 
+# Set these in the environment to override them. This is helpful for
+# development if you have file ownership problems because the user
+# in the container doesn't match the user on your host.
+SOCORRO_UID ?= 10001
+SOCORRO_GID ?= 10001
+
 default:
 	@echo "You need to specify a subcommand. See \"make help\" for options."
 	@exit 1
 
 help:
 	@echo "Socorro make rules:"
 	@echo ""
-	@echo "  build            - build docker containers for dev"
+	@echo "  build            - build docker containers"
```

Comment:
We use this rule for everything: dev, server environments, testing in CI, making waffles, etc. Figured I'd fix the help text while I was here.